site.yml

```diff
@@ -0,0 +1,51 @@
+site:
+  title: "Open Manufacturing Platform - Semantic Data Structuring Working Group"
+  url: https://openmanufacturingplatform.github.io/sds-documentation/
+  start_page: sds-documentation::index.adoc
+content:
+  sources:
+     - url: .
+       branches: HEAD
+       tags: '*'
+       start_path: src/docs
+     - url: https://github.com/OpenManufacturingPlatform/sds-bamm-aspect-meta-model
+       branches: HEAD
+       tags: '*'
+       start_path: src/docs
+     - url: https://github.com/OpenManufacturingPlatform/sds-sdk
+       branches: HEAD
+       tags: '*'
+       start_path: documentation/developer-guide
+ui:
+  bundle:
+    url: ./src/ui/ui-bundle.zip
```

Comment:
Can we not finally link against a released zip of the Antora UI?

Reply:
Good point, since we did a release of the SDS Antora UI, I will reference that instead.